Front-end/parquick/flow-typed/npm/@react-google-maps/api_vx.x.x.js

```diff
@@ -0,0 +1,74 @@
+// flow-typed signature: b33f2462b1671eb4a87167feee2411c7
+// flow-typed version: <<STUB>>/@react-google-maps/api_v2.12.1/flow_v0.182.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   '@react-google-maps/api'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module '@react-google-maps/api' {
+  declare module.exports: any;
+}
+
+/**
+ * We include stubs for each file inside this npm package in case you need to
+ * require those files directly. Feel free to delete any files that aren't
+ * needed.
+ */
+declare module '@react-google-maps/api/dist/cjs' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/dist/cjs.min' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/dist/esm' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/dist/esm.min' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/dist/umd' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/dist/umd.min' {
+  declare module.exports: any;
+}
+
+declare module '@react-google-maps/api/src/setup-tests' {
+  declare module.exports: any;
+}
+
+// Filename aliases
+declare module '@react-google-maps/api/dist/cjs.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/cjs'>;
+}
+declare module '@react-google-maps/api/dist/cjs.min.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/cjs.min'>;
+}
+declare module '@react-google-maps/api/dist/esm.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/esm'>;
+}
+declare module '@react-google-maps/api/dist/esm.min.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/esm.min'>;
+}
+declare module '@react-google-maps/api/dist/umd.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/umd'>;
+}
+declare module '@react-google-maps/api/dist/umd.min.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/dist/umd.min'>;
+}
+declare module '@react-google-maps/api/src/setup-tests.js' {
+  declare module.exports: $Exports<'@react-google-maps/api/src/setup-tests'>;
+}
```

Comment:
Things like this (adding modules, libraries, flow types, etc.) are better fit to be in their own pull request, just for future reference :)